dali/ft/daft.cpp

```diff
@@ -63,7 +63,7 @@ void CDistributedFileSystem::copy(IDistributedFile * from, IDistributedFile * to
     bool compressOutput = options->getPropBool("@compress");
     bool preserveCompression = options->getPropBool("@preserveCompression");
     LOG(MCdebugInfo, unknownJob, "DFS: compressInput:%d, compressOutput:%d, preserveCompression:%d ", compressInput, compressOutput, preserveCompression);
-    if (preserveCompression & compressInput)
+    if (preserveCompression && compressInput)
         sprayer->setTargetCompression(compressInput);
```

Comment:
Similar to prev. comment below, what if compressOutput = true and preserveCompression = false?

Reply:
Same as above.